otter/integration/tests/test_convergence.py

```diff
@@ -138,12 +150,12 @@ def test_reaction_to_oob_server_deletion(self):
         )
 
         self.scaling_policy = ScalingPolicy(
-            scale_by=1,
```

Comment:
See issue #1350 - This will be replaced with a to be implemented trigger convergence request.